third_party/proton/dialect/include/Conversion/ProtonGPUToLLVM/ProtonNvidiaGPUToLLVM/Passes.td

```diff
@@ -8,7 +8,7 @@ def ConvertProtonNvidiaGPUToLLVM : Pass<"convert-proton-nvidia-gpu-to-llvm", "ml
     let description = [{
         Convert ProtonGPU to LLVM using Nvidia-specific lowering patterns.
     }];
-    let constructor = "mlir::triton::proton::gpu::createConvertProtonNvidiaGPUToLLVMPass(80, 80)";
+    let constructor = "mlir::triton::proton::gpu::createConvertProtonNvidiaGPUToLLVMPass(\"\", 80, 80)";
```

Comment:
Why do we need a device arch string here?

Reply:
We don't need it for anything in the NV backend, I just needed to make the interface here consistent:
https://github.com/triton-lang/triton/blob/257c92c9036f25475774532c755585dd05d55f59/third_party/proton/proton/hooks/instrumentation.py#L148

https://github.com/triton-lang/triton/blob/257c92c9036f25475774532c755585dd05d55f59/third_party/proton/proton/hooks/instrumentation.py#L159-L167

but we use it in the AMD backend so having the interfaces be consistent seemed worthwhile.

Reply:
Why do we need to make the interface consistent?
They are different functions, right?
One is `add_convert_proton_nvidia_gpu_to_llvm`, another is `add_convert_proton_amd_gpu_to_llvm`

Reply:
Well the interface to the to_llvmir_passes lambda function has to be the same right b/c it is fed into the same interface here:
https://github.com/triton-lang/triton/blob/257c92c9036f25475774532c755585dd05d55f59/third_party/proton/proton/hooks/instrumentation.py#L146-L148

I can just break up that lambda function into nv and amd versions just seemed easier to include arch in both. But I don't feel strongly.

Reply:
I see what you mean but it's really not necessary to have an arch name at all for the nvidia backend. 
You can refer to the `passes.td` file in TritonAMDGPUToLLVM and TritonNvidiaGPUToLLVM.
For nvidia gpus, what we need is an integer number that represents the compute capability.

I think it's OK to have two different lambda functions for nv and amd

Reply:
Sounds good. Updated.